html/login.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="ko">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>로그인</title>
+    <link rel="icon" type="image/svg" href="/image/logo.svg" />
+    <link
+      rel="stylesheet"
+      as="style"
+      crossorigin
+      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard.min.css"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css"
+    />
+    <link rel="stylesheet" href="/css/auth.css" />
+  </head>
+
+  <body class="body_login">
+    <header>
+      <a href="/">
+        <img class="logo" src="/image/logo.svg" alt="판다마켓 로고" />
+      </a>
+      <a class="logo-title" href="/">
+        <img src="/image/logo_title.svg" alt="판다마켓 로고 글씨" />
+      </a>
```

Comment:
❗️ 수정요청
로고와 타이포가 같은 영역이니 하나의 a 태그로 묶는 것을 추천드립니다~

```suggestion
      <a href="/">
        <img class="logo" src="/image/logo.svg" alt="판다마켓 로고" />
        <img src="/image/logo_title.svg" alt="판다마켓 로고 글씨" />
      </a>
```